### DISCUSSION TITLE
你找到的策略

### DISCUSSION THREAD

**Discussion by Will150143** [ChiuAlfredo/options_day_data#49] [General]
可以把你找到的策略或觀察放這

**Comment by Will150143** [comment]
## 結算日當買方順勢策略
### 1.原因
- 時間價值剩沒多少能扣，最多權利金歸零
- Gamma會變大，指數稍微漲跌都會使權利金大增或大減
- 有機會翻倍，但因為敏感，一個回抽會回吐的很快
### 2.價性選擇
回測結果連結：https://reurl.cc/dyeW06
- 價平：對順勢策略來講，價平dealta不到1，所以回測起來績效其實不好
![價平績效曲線](https://github.com/user-attachments/assets/3ca60293-aa61-4e9a-85a9-a35757b345f1)
- 價內：回測結果較好。因為比較接近期貨特性，但又保有最多歸零的特性，只是成本變高，如果大幅度反向，還是比期貨會賠到無限大好
![與期貨比對](https://github.com/user-attachments/assets/0a49b861-4c89-4db9-9306-c94980092539)
- 價外：可以賺到穿價賺一波大的，但長期看起來，大部分時間勝率可能很低
*不宜做太價外，因為買權要靠漲或跌到某個價格才會獲利，太價外偏難
![價外兩檔績效曲線-1](https://github.com/user-attachments/assets/cdb44727-ac4f-415b-91a9-86a206a5a12a)
### 3.判斷入場時機
#### (1) 抓出星期二夜盤時間的高低點，在開盤之後觀察是否有突破
- 若突破高點，做買進買權
- 若突破低點，做買進賣權

#### (2) 用開盤價當基準（回測比用夜盤優）：漲超過多少或跌超過多少就直接追
### 4.停損
#### (1) 移動停損法
Ex：成本是100，設10%停損，跌到90出場，但漲到120，是120*0.9=108
#### (2) 均線
類似移動停損，價格越往上漲，均線隨著升高，停損點也升高
抓緊，出場點好，但可能過早下車
抓寬，可能可以吃到很大一段，但可能一個回檔導致獲利吐很多
#### (3) 不停損，放到結算（回測是這樣做的）
#### (4) 賺一倍時先出場一半保住成本
Ex：50點買兩口，漲到100先賣一口（維持零成本），剩下一口繼續拚

**Reply by Will150143** [reply]
## 延伸：雙買
預期短時間內會有波動率上升，有大漲跌的大行情出現，並且避免在權利金過高的時候進場，不然要需要加權漲更多才賺錢
### 優點
- 有限風險
### 缺點（風險）：
- 如果標的價格沒有在期限內達到預期目標，可能導致虧損。
- 如果標的價格波動較大，選擇權的價格也會劇烈波動，如果選擇權價格快速下跌，會造成買方遭受損失。
- 預期波動上升，權利金高漲，進去做買方失敗率其實非常高，等到時間價值回歸正常值，會導致選擇權價格暴跌。
### 注意事項
- 波動率：波動率越高，價格越高。因此應以短時間內有可能波動率上升的狀況進場，如果波動率已過高，進場失敗機率很高。
- 流動性：應該選擇流動性較高的資產，不要買賣太遠月都沒量之類的商品，以提高雙買策略的執行效率。
- 越價外比較便宜，不過越不容易到，所以獲利的機會比較低，但一旦穿過去，就會有以小博大翻倍的可觀獲利
- 價平or價內會比較貴，但比較容易穿過去，可是可能需要更大的行情讓獲利大於權利金

**Reply by Will150143** [reply]
## 延伸：用支撐壓力判斷入場
- 到支撐時（賣方開始防守）進場：賺回彈時的價差（逆勢）
- 跌到支撐買call
- 漲到壓力買put
*買方跟賣方勝率一樣：賣方守不住，一起賠
*價格很高時不如做期貨
- 價外拚倍率

**Comment by Will150143** [comment]
## 結算價計算
結算日當天，結算價格是以加權指數當天的13:00 ~ 13:25這段時間的價格，再加上最後13:30最後一筆的價格，用簡單算術平均取得價格，最終便是用這個價格定為這次契約的結算價。

實務上只需要看那一周要結算的周小台(第三個禮拜則是看大台或小台都可以)點數是多少，那麼結算價就差不多是多少。

也可以用選擇權價內履約價的買權加上它的權利金來推估結算價大約是多少。

不是看加權指數的收盤價，也不是在周契約結算時看月契約(例如大台)的點數

**Comment by Will150143** [comment]
別人分享的知識點
https://vocus.cc/article/66c09465fd897800014ef3e4
https://vocus.cc/article/66c21065fd89780001202f12

**Comment by TsuYiii** [comment]
## 結算日

### 倒莊

週三短線11點過後，看T字報價表，留意價平選擇權與價內一檔選擇權的點數，它的內涵價值與時間價值，看哪一邊篇多一點？偏多的地方代表賣方可以收到的租金最多，那就代表行情容易往另外一邊倒莊。（未回測）
出處：https://vocus.cc/article/64c3ba21fd897800015b7b92

我的想法：因倒莊大部分由大戶操作，所以透過觀察未平倉量時間價值與價格去觀察有無倒莊的可能性。
<img width="370" alt="價平時間價值圖" src="https://github.com/user-attachments/assets/052b70be-3e46-484c-8cb4-e561d71ad42b">
https://www.cmoney.tw/learn/course/1/topic/1423
